src/js/ui/analyse/interfaces.ts

```diff
@@ -251,4 +253,6 @@ export interface ExplorerData {
   recentGames?: Array<ExplorerGame>
   checkmate?: boolean
   stalemate?: boolean
+  variant_win?: boolean
+  variant_loss?: boolean
```

Comment:
This describes the tablebase JSON API. Unfortunately it already uses `insufficient_material`. But at least it's self consistent.